staging/src/k8s.io/client-go/discovery/cached/disk/cached_discovery_test.go

```diff
@@ -193,28 +193,26 @@ func TestOpenAPIDiskCache(t *testing.T) {
 				i++
 
 				_, err = v.Schema(contentType)
-				assert.NoError(t, err)
+				require.NoError(t, err)
 
 				path := "/openapi/v3/" + strings.TrimPrefix(k, "/")
 				assert.Equal(t, 1, fakeServer.RequestCounters[path])
 
 				// Ensure schema call is served from memory
 				_, err = v.Schema(contentType)
-				assert.NoError(t, err)
+				require.NoError(t, err)
 				assert.Equal(t, 1, fakeServer.RequestCounters[path])
 
 				client.Invalidate()
 
 				// Refetch the schema from a new openapi client to try to force a new
 				// http request
 				newPaths, err := client.OpenAPIV3().Paths()
-				if !assert.NoError(t, err) {
-					continue
-				}
+				require.NoError(t, err)
```

Comment:
Is this a correct conversion? The old logic looks like it flags an error and moves on to the next case without running further tests on this case. That does not appear to be what the new logic does.  IIUC it will abort this t.Run()?